### PR TITLE
Add openjpeg project

### PR DESCRIPTION
**Pull request by headshog**

### CODE REVIEW

File: projects/openjpeg/build.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+cmake -DCMAKE_BUILD_TYPE=Release ..
+make clean -s
+make -j$(nproc) -s
+cd ../tests/fuzzers
+
+# Build targets.
+if [[ $CONFIG = "sydr" || $CONFIG = "coverage" ]]
+then
+    $CC $CFLAGS -c -o /main.o /opt/StandaloneFuzzTargetMain.c
+    export MAIN_LINK="/main.o"
```
Comment (kobrineli) [inline projects/openjpeg/build.sh:77]: remove export here or upper. For Sydr it is repeated

File: projects/openjpeg/build.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    export LINK_FLAGS="$CFLAGS"
+    export SUFFIX="cov"
+fi
+
+# Build openjpeg.
+cd /openjpeg
+rm -rf build
+mkdir build
+cd build
+cmake -DCMAKE_BUILD_TYPE=Release ..
```
Comment (kobrineli) [inline projects/openjpeg/build.sh:68]: Isn't it needed to external pass compiler and flags to `cmake`?

File: projects/openjpeg/build.sh [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+then
+    $CC $CFLAGS -c -o /main.o /opt/StandaloneFuzzTargetMain.c
+    export MAIN_LINK="/main.o"
+fi
+
+FuzzerFiles=*.cpp
+for Filename in $FuzzerFiles; do
+    TargetName=$(basename $Filename .cpp)_$SUFFIX
+    $CXX $LINK_FLAGS -std=c++11 -I$SRC/src/lib/openjp2 -I$SRC/build/src/lib/openjp2 \
+        $Filename $* -o /$TargetName $SRC/build/bin/libopenjp2.a $MAIN_LINK -lm -lpthread
```
Comment (kobrineli) [inline projects/openjpeg/build.sh:84]: what `$*` is needed for?

**Review by kobrineli — CHANGES_REQUESTED** [review]

**Review by kobrineli — CHANGES_REQUESTED** [review]
Add README.md

File: projects/openjpeg/build.sh (hunk 2 above)
Comment (headshog) [reply projects/openjpeg/build.sh:67]: no need to pass flags

File: projects/openjpeg/build.sh (hunk 1 above)
Comment (headshog) [reply projects/openjpeg/build.sh]: removed

**Review by kobrineli — APPROVED** [review]

**Pull request merged**